payments/swish/after-payment.md

```diff
@@ -18,153 +18,12 @@ sidebar:
 
 {% include alert-development-section.md %}
 
-## Payment Resource
+## Operations
 
-The `payment` resource and all general sub-resources can be found in the
-[core payment resources][core-payment-resources] section.
-
-### Create Payment
```

Comment:
Why shouldn't we describe "Create Payment" in other features?

Reply:
Ref the two comments above. We are not sure where to place it.